util/compat_mcl.lua

```diff
@@ -0,0 +1,37 @@
+local mcl = minetest.get_modpath("mcl_core")
+
+logistica.sound_mod = mcl and mcl_sounds or default
+
+-- Returns a player's inventory formspec with the correct width and hotbar position for the current game
+function logistica.inventory_formspec(x,y)
+    local formspec
+    if exchangeclone.mcl then
```

Comment:
```suggestion
    if mcl then
```

Reply:
Whoops. I copied that from one of my mods.